docs/working-with-js-objects-and-values.md

```diff
@@ -137,8 +137,13 @@ advance, which can be helpful for prototyping or quickly generating JavaScript
 object literals.
 
 Melange provides some ways to create `Js.t` object values, as well as accessing
-the properties inside them. To create values, the `[%mel.obj]` extension is
-used, and the `##` infix operator allows to read from the object properties:
+the properties inside them.
+
+- To create values, use the <span class="text-ocaml">`[%mel.obj]` extension
```

Comment:
Added reasonml-specific text since formatting `let john = [%mel.obj { name : "john", age : 99 }];` in Reason syntax will convert it to just `let john = { "name": "john", "age": 99 }`